store-conf/conf/imap.log4j.properties

```diff
@@ -0,0 +1,53 @@
+# An example log4j configuration file that outputs to System.out.  The
+# output information consists of relative time, log level, thread
+# name, logger name, nested diagnostic context and the message in that
+# order.
+
+# For the general syntax of property based configuration files see the
+# documenation of org.apache.log4j.PropertyConfigurator.
+
+log4j.threshhold=OFF
+
+log4j.rootLogger=INFO,LOGFILE
+#log4j.rootLogger=INFO,LOGFILE,SYSLOG
+
+# Appender LOGFILE writes to the file "/opt/zimbra/log/imap-log4j.log".
+# Daily rolling policy with compressing the old log file while rotating!!
+# The archived log files location can be changed using FileNamePattern value
+log4j.appender.LOGFILE=org.apache.log4j.rolling.RollingFileAppender
+log4j.appender.LOGFILE.RollingPolicy=org.apache.log4j.rolling.TimeBasedRollingPolicy
+log4j.appender.LOGFILE.RollingPolicy.FileNamePattern=/opt/zimbra/log/imap-log4j.log.%d{yyyy-MM-dd}
+log4j.appender.LOGFILE.File=/opt/zimbra/log/imap-log4j.log
```

Comment:
Why not just `imapd.log` or `imapd-standalone.log`?  I think the log4j bit is self-evident.